packages/backend/src/asset/service.test.ts

```diff
@@ -115,6 +116,16 @@ describe('Asset Service', (): void => {
         AssetError.DuplicateAsset
       )
     })
+
+    test('Cannot create asset with scale > 255', async (): Promise<void> => {
+      const options = {
+        code: 'ABC',
+        scale: 256
+      }
+      await expect(assetService.create(options)).rejects.toThrow(
+        CheckViolationError
```

Comment:
I think I'm ok with throwing instead of returning an `AssetError` (https://github.com/interledger/rafiki/pull/1029#discussion_r1093353141) since this should already have been checked at all our entry points (Open Payments, admin api).
Unless there's some other route here I'm overlooking.

Reply:
That is what I was thinking. I just wrote the test to make sure that my check works.